docker/dev.yml

```diff
@@ -1,6 +1,18 @@
 version: "3.7"
 
 services:
+  datadog:
```

Comment:
we're no longer exposing our DD agent to the public internet, so we must spin one up on our own machine

Reply:
I had no luck using this agent. Do we need updated API keys?

Reply:
yes! i don't think action-devops has the latest key. Rafa is gonna update them, i didn't want to meddle.
i grabbed mine directly from DD. Key ending in `eda33`